map_org_names.yaml

```diff
@@ -14,7 +14,7 @@ mappings:
   - ['^[[:space:]]*absa[[:space:]]*group[[:space:]]*$', 'Absa Group Limited']
   - ['^[[:space:]]*absolute[[:space:]]*softare[[:space:]]*$', 'Absolute Software Corporation']
   - ['^[[:space:]]*accenture[[:space:]]*india[[:space:]]*pvt[[:space:]]*ltd[[:space:]]*$', 'Accenture Ltd.']
-  - ['^[[:space:]]*accord[[:space:]]*project[[:space:]]*$', 'Accord Project, LLC']
+  - ['^[[:space:]]*(accord[[:space:]]*project|Accord)[[:space:]]*$', 'Accord Project, LLC']
```

Comment:
RE is case insensitive, so please use only lower case letter in RE. Please apply to all other changes (MariaDB database is also case insensitive)

Reply:
@lukaszgryglicki I have fixed this.. Thanks :)